docs/about/whats-new/whats-new-0.7.0.ipynb

```diff
@@ -0,0 +1,466 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# What's new in scipp 0.7.0\n",
+    "\n",
+    "For a full list of changes see the [Release Notes](https://scipp.github.io/about/release-notes.html#v0-7-june-2021).\n",
+    "\n",
+    "## Various\n",
+    "\n",
+    "- Added `zeros_like`, `ones_like`, and `empty_like`.\n",
+    "- Added `linspace`, `logspace`, `geomspace`, and `arange`.\n",
+    "- Plotting supports `redraw()` method for updating existing plots with new data, without recreating the plot.\n",
+    "\n",
+    "## Performance\n",
+    "\n",
+    "- `sort` is now considerably faster for data with more rows.\n",
+    "- reduction operations such as `sum` and `mean` are now also multi-threaded and thus considerably faster.\n",
+    "\n",
+    "## Python-like shallow/deep copy mechanism\n",
+    "\n",
+    "The most significant change in this scipp release is a fundamental rework of all scipp data structures (variables, data arrays, and datasets).\n",
+    "These now behave mostly like nested Python objects, i.e., sub-objects are shared by default.\n",
+    "Previously there was no sharing mechanism and scipp always made deep-copies.\n",
+    "Some of the effects are exemplified in the following.\n",
+    "\n",
+    "### Variables\n",
+    "\n",
+    "For variables on their own, the new and old implementations mostly yield the same user experience.\n",
+    "Previously, views of variables, such as created when slicing a variable along a dimension, returned a different type &ndash; `VariableView` &ndash; which kept alive the original `Variable`.\n",
+    "This asymmetry is now gone.\n",
+    "Slices or other views of variables are now also of type `Variable`, and all views share ownership of the underlying data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import scipp as sc\n",
+    "if not sc.__version__.startswith('0.7'):\n",
+    "    print(f'This notebook was made for scipp-0.7 and will likely not work with your version ({sc.__version__}).')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If a variable refers only to a section of the underlying data buffer this is now indicated in the HTML view as part of the size on the top right, here *\"16 Bytes out of 96 Bytes\"*.\n",
```

Comment:
```suggestion
    "If a variable refers only to a section of the underlying data buffer this is now indicated in the HTML view as part of the size on the top left, here *\"16 Bytes out of 96 Bytes\"*.\n",
```

Reply:
Hmm, I meant to the right of the object name, will try to find some non-ambiguous wording.